### PR TITLE
handler implementation

### PR DESCRIPTION
**Pull request by sethvincent**
This isn't quite done, so I marked it as work-in-progress, but want to share where I'm at.

Input welcome! I don't spend a lot of time with python so I may have done wacky things on accident.

Open to ideas for other tests to run.

One thing that may need revision is the way we tell the handler what the directory of the task is. Here's how I'm doing that currently: https://github.com/cumulus-nasa/cumulus-message-adapter/pull/1/files#diff-1e58d0151b6843308790fd772efe84faR33

It should probably be similar across the node/java/python handlers?

### CODE REVIEW

File: tests/test_handler.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    handler(event, {}, handler_fn, handler_config) 
+
+class TestSledHandler(unittest.TestCase):
+
+    def test_working(self):
+        def test_fn(event, context):
+            return event
+
+        response = run_test_handler(create_event(), test_fn)
+        print response
```
Comment (abarciauskas-bgse) [inline tests/test_handler.py]: Probably we don't want this print statement here?

File: tests/test_handler.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        self.assertTrue(response)
```
Comment (abarciauskas-bgse) [inline tests/test_handler.py]: I know this is WIP but probably we want to assert the actual value of the response matches our expectations?

**Comment by sethvincent** [comment]
@jennyhliu Currently I'm seeing this error and haven't figured out yet if it's something I messed up or if it's related to that jsonpath_ng method that isn't implemented, or something else.

```
ERROR: test_working (__main__.TestSledHandler)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/sdv/workspace/cumulus-nasa/cumulus-sled/tests/test_handler.py", line 38, in test_working
    response = run_test_handler(create_event(), test_fn)
  File "/Users/sdv/workspace/cumulus-nasa/cumulus-sled/tests/test_handler.py", line 30, in run_test_handler
    handler(event, {}, handler_fn, handler_config)
  File "cumulus_sled/handler.py", line 65, in handler
    nested_event = msg.loadNestedEvent(full_event, context)
  File "cumulus_sled/message.py", line 262, in loadNestedEvent
    finalPayload = self.__resolveInput(event, config);
  File "cumulus_sled/message.py", line 251, in __resolveInput
    return event.payload;
AttributeError: 'dict' object has no attribute 'payload'
```

File: tests/test_handler.py (hunk 2 above)
Comment (sethvincent) [reply tests/test_handler.py]: Definitely :+1:

**Comment by sethvincent** [comment]
@jennyhliu @abarciauskas-bgse This is ready for another look. I merged in updates from the master branch and made changes so it works the nosetests/circleci.

Open to ideas for other tests to run.

One thing that may need revision is the way we tell the handler what the directory of the task is. Here's how I'm doing that currently: https://github.com/cumulus-nasa/cumulus-message-adapter/pull/1/files#diff-1e58d0151b6843308790fd772efe84faR33

It should probably be similar across the node/java/python handlers?

**Comment by sethvincent** [comment]
Closing this PR in favor of putting the handler code in a separate repo: https://github.com/cumulus-nasa/cumulus-message-adapter-python

**Pull request closed without merging**